src/main/java/sirius/db/mongo/Finder.java

```diff
@@ -371,16 +373,35 @@ public long countIn(Class<?> type) {
      * Counts the number of documents in the result of the given query.
      * <p>
      * Note that limits are ignored for this query.
+     * If the there are no filters in this query, an estimate is returned instead.
      *
      * @param collection the collection to search in
      * @return the number of documents found
      */
     public long countIn(String collection) {
+        return countIn(collection, false, 0);
+    }
+
+    /**
+     * Counts the number of documents in the result of the given query.
+     * <p>
+     * Note that limits are ignored for this query.
+     * If the there are no filters in this query and forceAccurate is false, a pre-counted estimate is returned instead.
+     *
+     * @param collection    the collection to search in
+     * @param forceAccurate always count the actual query using countDocuments
```

Comment:
```suggestion
     * @param forceAccurate if set to &lt;tt&gt;true&lt;/tt&gt; we'll never use &lt;b&gt;estimatedDocumentCount&lt;b&gt; which is way more efficient but might return wrong values in case a cluster is active which had experienced an unclean shutdown.
```